schemas/log.yml

```
@@ -35,3 +35,11 @@
 
         This field is not indexed and doc_values are disabled so it can't be
         queried but the value can be retrieved from `_source`.
+
+    - name: logger
+      level: core
+      type: keyword
+      example: org.elasticsearch.bootstrap.Bootstrap
+      short: Name of the logger.
+      description: >
+        The name of the logger, usually the name of the class which initialized the logger.
```

I would flesh the description out a little more, to try to clarify what scenarios this is meant to be used in. Could be something like this:

```suggestion
        The name of the logger inside an application. This is usually the name of the class which initialized the logger, or can be a custom name. 
```